cmd/machine-config-daemon/pivot.sh

```diff
@@ -1,2 +1,2 @@
 #!/usr/bin/sh
-exec /usr/libexec/machine-config-daemon pivot "$@"
+exec /usr/local/machine-config-daemon pivot "$@"
```

Comment:
`bin` is missing in the m-c-d path

Reply:
Updated. Is it being used at all in 4.5?

Reply:
Don't think so but I believe this is mainly there to maintain compatibility with cluster when [pivot ](https://github.com/openshift/pivot) was maintained separately. @cgwalters  would have more insights.

Reply:
I believe this was simply for humans to use interactively via `oc debug node/` or `ssh`.  It's OK if we remove it I guess, but it'd also be nice to keep it working.